initgeneration/init.sh.test-sample

```diff
@@ -135,6 +154,7 @@ createConfigurationFiles() {
 		if [[ "${host_tenant}" != "-" ]]; then
 			if [[ "abc12345" == "${host_tenant}" ]]; then
 				writeConfK8To ${container_conf_file}
+				writeConfEnrichmentTo "/var/lib/dynatrace/enrichment" "dt_metadata"
```

Comment:
maybe dumb question:
do you only want to do `enrichment` in case of cloudNativeFullstack ?
because  `${host_Tenant}` is only set to anything in case of cloudNativeFullstack and you could have activegates without cloudNativeFullstack

Reply:
Right, enrichment configuration should be created in both CloudNativeFullStack and ApplicationMonitoring mode.